### ISSUE TITLE
NOT ISSUE - QUESTION: Simple slider & bind to list of existing values

### ISSUE THREAD

**Issue by kuatroka** [techniq/svelte-ux#147]
It's a question or maybe a feature request.

Is there a way to make the RangeSlider, just a slider with only one point of selection along the track?

And the follow up - If the answer to the first question is yes, then is there a way to pass a list of values to the slider instead of min/max? The reason is the following:
I got an array of quarters ['200Q1', '2016Q2', .....etc] and I want to use it in a slider with a bar chart for example. I would slide the slider and would select a quarter to show the data for. 
There is also a filter to choose an entity for which a different array of quarters will be used for the slider and the array is not always filled with consecutive quarters, some times some quarters are missing, so I can't have a min/max.

Is there a way to bind my own/custom array of values in my predefined order to this Single point Slider? Thanks.

**Comment by techniq** [comment]
Hey @kuatroka 👋

At the moment there is not a way to select a single value using `RangeSlider`, although I have a lot of plans for the component (see #54).  `RangeSlider` is going to become `MultiRangeInput`, and a new `RangeInput` that supports a single value (and like using a native `<input type="range">` (but styled) will be used).  `MultiRangeInput` also needs some work TBH.

One thought you could do is use `<input type="range">` with a `<datalist>`.  I've experimented with this some ([REPL](https://svelte.dev/repl/9e8b2964cfbc4751b0096d9d262cec44?version=4.2.8)), but it needs refined.   See also [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range) doc.

Also, I don't know if you've seen, but [Range Slider Pips](https://simeydotme.github.io/svelte-range-slider-pips/en/introduction/) looks great (I haven't used it personally, but it looks really well polished).

A slightly different idea, instead of a range, you might want to consider `MenuField` with the [stepper](https://svelte-ux.techniq.dev/docs/components/MenuField#stepper) prop.  I use it on the LayerChart in a few places, including [here](https://www.layerchart.com/docs/components/MotionPath) (see path/curve fields)

So in summary,
- It's not currently possible, but likely will be in the future
- You might want to trying using `<input type="range">` with `<datalist>`
- or [Range Slider Pips](https://simeydotme.github.io/svelte-range-slider-pips/en/introduction/)
- or a `MenuField` with [stepper](https://svelte-ux.techniq.dev/docs/components/MenuField#stepper)

**Comment by kuatroka** [comment]
Got it, also very nice. Yes, the stepper function would be nice too there. With the `options` prop you made if very simple pass a custom list of values. Exactly what I need. Thanks a lot.
